LICENSE

```diff
@@ -1,190 +1,32 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+Pachyderm License
```

Comment:
Should we call this Community License since that's how we refer to it everywhere else?

Reply:
I checked everyone I could think of who changed their license:
Mongo - sspl
Redis - 3 clause BSD
Confluent - Confluent Community License Agreement
Sentry -3 clause BSD 
Cockroach - Business Source License 1.1 (interesting Mariadb has actually trademarked this license)
Elastic - ELASTIC LICENSE AGREEMENT
Timescale - Timescale License (TSL)
Neo4j - Commons Clause
Dgraph - Dgraph Community License Agreement
etc.

I don't necessarily see a patter per-say, but I'll change it just to keep things consistent. And just to make sure... the lawyers won't have any issues with us changing the title?

Reply:
fixed in next commit